packages/mobile/src/escrow/reducer.ts

```diff
@@ -1,11 +1,14 @@
 import { Actions, ActionTypes, EscrowedPayment } from 'src/escrow/actions'
+import { RootState } from 'src/redux/reducers'
 
 export interface State {
   sentEscrowedPayments: EscrowedPayment[]
+  suggestedFee: string
 }
 
 export const initialState = {
   sentEscrowedPayments: [],
+  suggestedFee: '',
```

Comment:
We try to avoid using empty strings, better to use `null` to indicate something doesn't have a value yet. This will require setting the type to `string | null` above